src/lib.rs

```diff
@@ -740,15 +736,27 @@ impl SystemHandle {
 
             registry
                 .drain(..)
-                .rev()
                 .enumerate()
```

Comment:
@strohel Just curious what the justification was for switching the order? This way the indices are set by the original order I guess?